src/components/SvgIcon.js

```diff
@@ -17,13 +17,13 @@ import { useLayoutEffect, useRef } from 'preact/hooks';
  *
  * @type {IconMap}
  */
-let iconRegistry = {};
+const iconRegistry = new Map();
 
 /**
  * @typedef SvgIconProps
- * @prop {string} name - The name of the icon to display.
+ * @prop {string|symbol} name - The name of the icon to display.
```

Comment:
Do you support `string` to transition from the old `registerIcons` to the new `registerIcon`?